controllers/suite_test.go

```diff
@@ -152,6 +154,10 @@ func setup() error {
 	// +kubebuilder:scaffold:scheme
 	k8sManager, err = ctrl.NewManager(cfg, ctrl.Options{
 		Scheme: scheme.Scheme,
+		CertDir:            testEnv.WebhookInstallOptions.LocalServingCertDir,
+		Port:               testEnv.WebhookInstallOptions.LocalServingPort,
+		// TODO: Do we need to do the below - we have it in k8s-infra but don't seem to need it here.
```

Comment:
this is because we run multiple instances of envtest. if you're just running one then you won't get port conflicts

Reply:
Thanks for the clarification, have removed this